builtins/builtins.go

```diff
@@ -34,6 +35,15 @@ func Int2Char(args ...runtime.Object) runtime.Object {
 	return runtime.Errorf("Argument is out of range. Range is from 0 to 127. Int = %s", n.String())
 }
 
+func Str2Int(args ...runtime.Object) runtime.Object {
+	rstring := args[0].(*runtime.String)
```

Comment:
It customary in Go to use [short variables names](https://github.com/golang/go/wiki/CodeReviewComments#variable-names). I think `rstring` would be a good candidate:
```suggestion
	s := args[0].(*runtime.String)
```